tests/dummy/app/routes/components/fixed-top-nav-fixed-width-body-tabs/template.hbs

```diff
@@ -0,0 +1,41 @@
+<h2>Fixed Top Nav w/ Fixed Width Body Tabs</h2>
+
+{{!- BEGIN-SNIPPET fixed-top-nav-fixed-width-body-tabs }}
+<page class="example-layout page1">
+  <hbox fit justify="between" class="blue header">
```

Comment:
I'm okay with that, assuming they work. I didn't write these examples, @optikalefx did. Side note @optikalefx you need to update your git email, the commits you made don't seem to be linked to your github account.